classifier/run.py

```diff
@@ -0,0 +1,263 @@
+
+'''
+Script to train and test litter classifier.
+'''
+import torch
+import torchvision
+import torchvision.transforms as transforms
+
+import torch.nn as nn
+import torch.optim as optim
+from torch.utils.data.sampler import SubsetRandomSampler
+from torch.autograd import Variable
+
+import matplotlib.pyplot as plt
+import numpy as np
+import os
+import argparse
+
+
+def get_args_parser():
+    parser = argparse.ArgumentParser(
+        'Train and test network for classification task')
+    parser.add_argument('--data_img',
+                        help='path to directory with subdirectories with images',
+                        type=str)
+    parser.add_argument('--out',
+                        help='path to main directory with checkpoints',
+                        type=str)
+    parser.add_argument('--mode', default='train',
+                        help='type of procedure: test or train',
+                        choices=['train', 'test'],
+                        type=str)
+    parser.add_argument('--name', default='test.png',
+                        help='path to save test images', type=str)
+    parser.add_argument('--num', help='number of images to display',
+                        default=5, type=int)
+    parser.add_argument('--device', help='specify device to use',
+                        default="cuda:0", type=str)
+    parser.add_argument('--epochs', default=30, type=int)
+    parser.add_argument('--lr', default=0.001, type=float)
+    parser.add_argument('--momentum', default=0.9, type=float)
+    return parser
+
+
+def make_weights_for_balanced_classes(images, nclasses=6):
```

Comment:
you can use collections.Counter in future

Reply:
Ok, I will check this, thanks ;-)